src/styles/abstracts/_variables.scss

```diff
@@ -3,4 +3,8 @@ $color-lightgray-font: #bdbdbd;
 $color-darkgray: #333;
 $color-black: #000;
 
+$header-height: 50px;
 $slots-height: 60px;
+$backspace-width: 50px;
+$backspace-height: 50px;
```

Comment:
why? I use it in other components as well.

Reply:
I didn't notice that!

I think it would be cleaner to have component's styles that aren't so tightly coupled and don't depend on each other so much

Reply:
in this case they do depend on each other 🤷‍♀️ unless you have some other solution on your mind (the goal is to center slots on a bigger screen having an extra column on the right).

Reply:
ideas:
* absolute positioning of `Backspace` component
* changing the goal: center slots+backspace

(I'm just reading the code, maybe I'm not aware of some circumstances you know well because you're coding it)

Reply:
>     * absolute positioning of `Backspace` component
it's not that simple as Backspace is a child of the Game, not Slots (and I would keep it this way)

>     * changing the goal: center slots+backspace
then it would hurt MY eyes 😄 

I'd say let's leave it, hm?

Reply:
You're the President, I'm National Security Advisor!